pymc/logprob/transform_value.py

```diff
@@ -119,12 +119,6 @@ def transformed_value_logprob(op, values, *rv_outs, use_jacobian=True, **kwargs)
         if log_jac_det.ndim < logp.ndim:
             diff_ndims = logp.ndim - log_jac_det.ndim
             logp = logp.sum(axis=np.arange(-diff_ndims, 0))
-        # This case is sometimes, but not always, trivial to accomodate depending on the "space rank" of the
-        # multivariate distribution. See https://proceedings.mlr.press/v130/radul21a.html
-        elif log_jac_det.ndim > logp.ndim:
-            raise NotImplementedError(
-                f"Univariate transform {transform} cannot be applied to multivariate {rv_op}"
-            )
         else:
```

Comment:
We should actually always run this last else check

Reply:
Fixed in https://github.com/pymc-devs/pymc/pull/7023/commits/0084e030bac14cc81cd6c7fa34956ca869d97bae